package.json

```
@@ -1,15 +1,31 @@
 {
-  "name": "comete-design-tokens",
-  "version": "1.0.0",
+  "name": "@aexae/design-tokens",
+  "version": "0.0.1",
   "type": "module",
+  "files": [
```

Une contrainte sur la version de node pour être cohérent avec les autres projets ?
```suggestion
  "engines" {
    "node": ">=22"
  },
  "files": [
```